subgraphs/uniswap-forks/config/_networkConfig.ts

```diff
@@ -7,6 +7,7 @@ import { toBytesArray } from "../src/common/utils/utils";
 export namespace Protocol {
```

Comment:
I wonder if this can be done more cleanly using inheritance: https://www.typescriptlang.org/docs/handbook/classes.html

(same applies to the Uniswap-v3 code)

Reply:
Also, I'd break this file into multiple files, one for each fork